src/components/in-app-messenger/_variables.scss

```diff
@@ -0,0 +1,16 @@
+$banner-height: 40px;
```

Comment:
Is `in-app-messenger` relevant to ui elements? Or is it just the styles that we need?

Reply:
I moved this file to the base-table folder where it's being used, so there's no longer an in-app-messenger folder.  Should I merge these styles into another file in the base-table folder, or keep them in the separate file they're already in?

Reply:
Found these styles were never actually used in our code.  Removed the file.